guesser.rb

```diff
@@ -0,0 +1,45 @@
+puts "Think of a number from 1 to 100"
+puts "I try to guess your number"
+puts "Answer only 'too low', 'too high' or 'correct'"
+number_guess = 0
+lowest = 1
+highest = 100
+
+loop do
+    number_guess += 1
+    guess = (lowest + highest) / 2
+    puts "Is your number #{guess}?"
+    answer = gets.chomp.downcase
+    if answer == "too low"
+        if guess >= highest
+            puts "You are lying! The guess can't be too low."
+            break
```

Comment:
The final requirement of this program is:

> Once it guesses correctly, or figures out you are lying, it should prompt you to play again, exiting if you decline.

However, this doesn't appear to prompt me to play again, as it just exits as soon as it says that I'm lying.

Reply:
Thank you! I missed this requirement